lms/djangoapps/certificates/apis/v0/views.py

```diff
@@ -0,0 +1,119 @@
+
+        **GET Parameters**
+
+            A GET request must include the following parameters.
+
+            * username: A string representation of an user's username.
+            * course_id: A string representation of a Course ID.
+
+        **GET Response Values**
+
+            If the request for information about the Certificate is successful, an HTTP 200 "OK" response
+            is returned.
+
+            The HTTP 200 response has the following values.
+
+            * username: A string representation of an user's username passed in the request.
+
+            * course_id: A string representation of a Course ID.
+
+            * certificate_type: A string representation of the certificate type.
+                Can be honor|verified|professional
+
+            * status: A string representation of the certificate status.
+
+            * download_url: A string representation of the certificate pdf url.
+
+            * grade: A string representation of a float for the user's course grade.
+
+        **Example GET Response**
+
+            {
+                "username": "bob",
+                "course_id": "edX/DemoX/Demo_Course",
+                "certificate_type": "verified",
+                "status": "downloadable",
+                "download_url": "http://www.example.com/cert.pdf",
+                "grade": "0.98"
+            }
+    """
+
+    authentication_classes = (OAuth2Authentication, SessionAuthentication,)
+    permission_classes = (IsAuthenticated, permissions.IsUserInUrlOrStaff)
+
+    def get(self, request, username, course_id):
+        """
+        Gets a certificate information.
+
+        Args:
+            request (Request): Django request object.
+            username (string): URI element specifying the user's username.
+            course_id (string): URI element specifying the course location.
+
+        Return:
+            A JSON serialized representation of the certificate.
+        """
+        try:
+            course_key = CourseKey.from_string(course_id)
+        except InvalidKeyError:
+            log.info('Course ID string "%s" is not valid', course_id)
+            return Response(
+                status=404,
+                data={
+                    'error_code': 'Course ID string "%s" is not valid'.format(course_id)
+                }
+            )
+
+        user_cert = get_certificate_for_user(username=username, course_key=course_key)
+        if user_cert is None:
+            return Response(
+                status=404,
+                data={
+                    'error_code': ('No certificate available '
+                                   'for user "{}" for the course "{}"').format(
+                        username, course_id)
+                }
+            )
+        return Response(
+            {
+                "username": user_cert.get('username'),
+                "course_id": user_cert.get('course_key'),
```

Comment:
actually the reason for the change of name is because in the new API edx is developing (and that will replace this one) the fields are called in this way.